### PR TITLE
Endret litt på flyten i KlagebehandlingDistribusjonService, slik at d…

### PR DESCRIPTION
**Pull request by oyvind-wedoe**
…et alltid er oppdatert vedtaks-objekt som brukes.

Endret test til å sjekke forventet output gitt input, i stedet for å teste kall fra under-tjenester.

### CODE REVIEW

File: src/main/kotlin/no/nav/klage/oppgave/service/distribusjon/KlagebehandlingDistribusjonService.kt [hunk 1]
```diff
@@ -42,12 +42,12 @@ class KlagebehandlingDistribusjonService(
                             logger.debug("Vedtak ${vedtak.id} i klagebehandling $klagebehandlingId er ikke distribuert til brevmottaker ${brevMottaker.id}")
 
                             klagebehandling =
-                                lagJournalpostKopierForSekundaereMottakere(klagebehandling, vedtakMedOppdaterteBrevmottakere, brevMottaker)
+                                lagJournalpostKopierForSekundaereMottakere(klagebehandling, vedtak.id, brevMottaker)
```
Comment (flexable777) [inline src/main/kotlin/no/nav/klage/oppgave/service/distribusjon/KlagebehandlingDistribusjonService.kt:45]: Ser at du byttet ut nesten alle vedtak til vedtak.id. Lurer på om man burde gjort det for `lagBrevmottakere` også, og kanskje da mappe vedtak til vedtakId i `forEach`-en?

File: src/test/kotlin/no/nav/klage/oppgave/service/distribusjon/KlagebehandlingDistribusjonServiceTest.kt [hunk 2]
```diff
@@ -144,26 +157,25 @@ internal class KlagebehandlingDistribusjonServiceTest {
     }
 
     @Test
-    fun `dokumentdistribusjon kalles en gang for vedtak med en mottager`() {
+    fun `distribusjon av klagebehandling fører til dokdistReferanse, ferdig distribuert vedtak og avsluttet klagebehandling`() {
```
Comment (flexable777) [inline src/test/kotlin/no/nav/klage/oppgave/service/distribusjon/KlagebehandlingDistribusjonServiceTest.kt:160]: Kan sikkert starte med dette ("happy path"). Men kanskje hadde vært kult å sjekke ulike corner cases også. Men det kan man utvide med over tid.

**Review by flexable777 — APPROVED** [review]
